### DISCUSSION TITLE
The SPOC Jupyter Book

### DISCUSSION THREAD

**Discussion by amandawhitmire** [sul-dlss-labs/spoc#29] [General]
This will be my first time contributing to a Jupyter book, so I want to be sure we are all on the same page (see what I did there) about the content. There needs to be a useful balance between offering background on the Miller Library/Hopkins Marine Station Student Papers Project and the Stanford Libraries' AI component of building out tools to extract species occurrences from the texts. As I help build out the book content, I welcome feedback on my contributions so that we can iterate on finding that right balance. Looking forward to it!

**Comment by jermnelson** [comment]
@amandawhitmire I agree! On the flip side, please let me know if there is AI content that may be too technical for inclusion (particularly in some of the Jupyter notebooks)
